lib/routes/tweets.js

```diff
@@ -0,0 +1,55 @@
+const { Router } = require('express');
+const Tweet = require('../models/Tweet');
```

Comment:
```suggestion
const User = require('../models/User);
const Tweet = require('../models/Tweet');
```
I posted some Tweets to your server and those worked fine, but get requests didn't work.  Your User model needed to be instantiated here.